frontend/src/components/Pages/MissionHistoryPage/MissionHistoryView.tsx

```diff
@@ -33,31 +55,60 @@ export function MissionHistoryView({ refreshInterval }: RefreshProps) {
     const [isLoading, setIsLoading] = useState<boolean>(true)
     const [isResettingPage, setIsResettingPage] = useState<boolean>(false)
 
-    const {
-        page,
-        switchPage,
-        missionName,
-        statuses,
-        robotName,
-        tagId,
-        inspectionTypes,
-        minStartTime,
-        maxStartTime,
-        minEndTime,
-        maxEndTime,
-    } = useMissionFilterContext()
+    const { page, switchPage, filterState, filterIsSet, filterFunctions, filterError, clearFilterError } =
+        useMissionFilterContext()
+
+    const FilterErrorDialog = () => {
+        return (
+            <>
+                <Dialog open={filterError !== ''} isDismissable onClose={() => clearFilterError()}>
+                    <Dialog.Header>
+                        <Dialog.Title>Dismissable dialog</Dialog.Title>
+                    </Dialog.Header>
+                    <Dialog.CustomContent>
+                        <Typography variant="body_short">Closes dialog on click outside and escape key.</Typography>
```

Comment:
Shouldn't filterError be used as the content?

Reply:
yes, absolutely